app/helpers/blacklight/catalog_helper_behavior_decorator.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# OVERRIDE blacklight 6 to allow full url for show links for shared search thumbnail
+Blacklight::CatalogHelperBehavior.module_eval do
```

Comment:
@laritakr Did this file come from somewhere? This works but we usually like to override files using the decorator pattern: https://playbook-staging.notch8.com/en/dev/ruby/decorators-and-class-eval

Reply:
It would look like the following: 

```
# frozen_string_literal: true

# OVERRIDE blacklight 6 to allow full url for show links for shared search thumbnail
module Blacklight
  module CatalogHelperBehavior
    def render_thumbnail_tag(document, image_options = {}, url_options = {})
      value = if blacklight_config.view_config(document_index_view_type).thumbnail_method
                send(blacklight_config.view_config(document_index_view_type).thumbnail_method, document, image_options)
              elsif blacklight_config.view_config(document_index_view_type).thumbnail_field
                url = thumbnail_url(document)
                image_tag url, image_options if url.present?
              end

      # rubocop:disable Style/GuardClause
      if value
        if url_options == false
          # rubocop:disable Metrics/LineLength
          Deprecation.warn(self, "passing false as the second argument to render_thumbnail_tag is deprecated. Use suppress_link: true instead. This behavior will be removed in Blacklight 7")
          # rubocop:enable Metrics/LineLength
          url_options = { suppress_link: true }
        end
        if url_options[:suppress_link]
          value
        elsif url_options[:full_url]
          link_to generate_work_url(document.to_h, request) do
            value
          end
        else
          link_to_document document, value, url_options
        end
      end
      # rubocop:enable Style/GuardClause
    end
  end
end

Blacklight::CatalogHelperBehavior.prepend(Blacklight::CatalogHelperBehaviorDecorator)
```